### DISCUSSION TITLE
41755833-971A-7F24-F769-C9332075DD00

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#55] [General]
# Proxima Centauri

Proxima Centauri is the nearest star to the Sun. Located only 4.2 light-years away, this red dwarf star has a total of two confirmed rocky planets orbiting around it, one of which are even considered as a potentially habitable world.

https://celestia.mobi/addon?item=41755833-971A-7F24-F769-C9332075DD00

**Comment by xpple-o-O** [comment]
For some reason, it proxima orbits alpha centauri but the planets arent there! https://imgur.com/a/vwRuRAu

**Reply by Celestiabit** [reply]
Nice mod i will say :)

**Reply by TheLostProbe** [reply]
are you on desktop or mobile? if you're on desktop, did you install the add-on correctly? and if you're on mobile, did you restart Celestia after installing the add-on?

**Reply by exocritic5457** [reply]
Any plans for the alf Cen AB barycenter to orbit around the barycenter between itself and Proxima, or was it intentionally left out?

**Comment by blackfloweria** [comment]
Well the textures on Mobile is really glitchy

**Comment by EfremAlvares** [comment]
Dis ap is perfect to spase lerners

**Comment by JohnJake14** [comment]
Proxima C should be added to Hypothetical and Disproven

**Reply by DaveBowman2001** [reply]
Already moved as an independent add-on since [July 24, 2025](https://celestia.mobi/resources/item?item=351C6C97-7C6A-4193-8243-15449FFA87A9). It isn't fully disproven yet, rather it's just inconclusive as of two studies published in 2025 ([Artigau](https://arxiv.org/abs/2207.13524) and [Mascareno](https://arxiv.org/abs/2507.21751))
